### PR TITLE
whistle 2.4.4

### PR DESCRIPTION
**Pull request by echopi**
Created with `brew bump-formula-pr`.

### CODE REVIEW

File: Formula/whistle.rb [hunk 1]
```diff
@@ -3,9 +3,9 @@
 class Whistle < Formula
   desc "HTTP, HTTP2, HTTPS, Websocket debugging proxy"
   homepage "https://github.com/avwo/whistle"
-  url "https://registry.npmjs.org/whistle/-/whistle-2.4.2.tgz"
+  url "https://registry.npmjs.org/whistle/-/whistle-2.4.4.tgz"
   # version "2.4.2"
```
Comment (bayandin) [inline Formula/whistle.rb]: Could you delete this comment, please?

Comment (echopi) [reply Formula/whistle.rb]: Sure! I'll make a pr later.

Comment (echopi) [reply Formula/whistle.rb]: done

Comment (bayandin) [reply Formula/whistle.rb]: These trivial changes don't require a separate PR, let's add them into this PR. I would say let's add them to the commit with version bump.

Comment (echopi) [reply Formula/whistle.rb]: Got it. Thank you.

**Pull request closed without merging**